HashRegistrarSimplified.sol

```diff
@@ -334,11 +334,11 @@ contract Registrar {
         } else if(auctionState != Mode.Reveal) {
             // Invalid phase
             throw;
-        } else if (_value < minPrice) {
+        } else if (actualValue < minPrice) {
```

Comment:
We used to do that, but we didn't see any harm in this more lenient approach. The punishment for an invalid bid is losing it all, so we wanted to err on the side of not throwing away peoples' funds if there's no attack it enables.